src/pal/tests/palsuite/paltestlist.txt

```diff
@@ -532,6 +532,7 @@ file_io/DeleteFileA/test1/paltest_deletefilea_test1
 file_io/DeleteFileW/test1/paltest_deletefilew_test1
 file_io/errorpathnotfound/test2/paltest_errorpathnotfound_test2
 file_io/errorpathnotfound/test3/paltest_errorpathnotfound_test3
+file_io/filecanonicalizepath/paltest_filecanonicalizepath_test1
```

Comment:
Looks like the CI build failed on Linux; I expect it's because this capitalization doesn't match that of the FILECanonicalizePath directory.